app/views/events/edit.html.erb

```diff
@@ -1,3 +1,3 @@
 <h1>Edit event</h1>
 <%= render 'form', event: @event %>
-<%= link_to 'Back', event_path(@event) %>
+<%= button_to 'Back', events_path, method: :get, class: 'btn-3 ml-5' %>
```

Comment:
Use link_to for get requests, and button_to for delete and post requests

Reply:
Done!